docs/use-cases/experiments.md

```diff
@@ -0,0 +1,316 @@
+|--------------------------|----------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+| Purpose                  | Compare two or more variants of a single element               | Simultaneously test multiple elements and variations                                           |
+| Variants                 | Two or more variants (Control and Treatment)                   | Multiple variants for each element being tested                                                |
+| Scope                    | Focuses on one element at a time                               | Tests combinations of elements and their variations                                            |
+| Complexity               | Relatively simpler to set up and analyze                       | and analyze	More complex to set up and analyze                                                 |
+| Statistical Significance | Typically requires fewer samples to achieve significance       | Requires larger sample sizes to achieve significance                                           |
+| Insights                 | Provides insights into the impact of individual changes        | Provides insights into the interaction between changes                                         |
+| Test Duration            | Generally shorter duration                                     | Often requires longer duration to obtain reliable results                                      |
+| Examples                 | Ideal for testing isolated changes, UI tweaks, copy variations | Useful for testing larger-scale changes, page redesigns, interaction between multiple elements |
+
+## Your application
+
+For this guide, let's say your application consists of a landing page containing these elements:
+
+- **Hero section**: The main section of the landing page, which includes:
+  - headline
+  - subheading, and
+  - call-to-action (CTA) button
+
+We now want to run both A/B Tests and Multivariate Tests using Featurevisor.
+
+## A/B Test on CTA button
+
+Let's say we want to run an A/B Test on the CTA button in the Hero section of your landing page.
+
+The two variations for a simple A/B test experiment would be:
+
+- **control**: The original CTA button with the text "Sign up"
+- **treatment**: The new CTA button with the text "Get started"
+
+We can express that in Featurevisor as follows:
+
+```yml
+# features/ctaButton.yml
+description: CTA button
+tags:
+  - all
+
+bucketBy: deviceId
+
+defaultVariation: control
+
+variations:
+  - value: control
+    description: Original CTA button
+    weight: 50
+
+  - value: treatment
+    description: New CTA button that we want to test
+    weight: 50
+
+environments:
+  production:
+    rules:
+      - key: "1"
+        segments: "*" # everyone
```

Comment:
Do you want to show/link here how someone can target different segments?

Reply:
added a new code block showing example of targeting specific segments under this sub-heading's section.